webapp/src/utils/new-format-objects.js

```diff
@@ -105,37 +57,144 @@ export const newDataFormatPercentByElection = (
 
   return percentByElectionData.map(data => ({
     name: data?.delegate_payer || data?.recipient,
-    [`EOS_${uppercaseCategory}`]: Number(data[`eos_${category}`]) * 100,
-    [`EOS_UN${uppercaseCategory}`]: Number(data[`eos_un${category}`]) * 100,
-    [`USD_${uppercaseCategory}`]: Number(data[`usd_${category}`]) * 100,
-    [`USD_UN${uppercaseCategory}`]: Number(data[`usd_un${category}`]) * 100
+    [`EOS_${uppercaseCategory}_PERCENT`]: Number(data[`eos_${category}`]) * 100,
+    [`EOS_UN${uppercaseCategory}_PERCENT`]:
+      Number(data[`eos_un${category}`]) * 100,
+    [`USD_${uppercaseCategory}_PERCENT`]: Number(data[`usd_${category}`]) * 100,
+    [`USD_UN${uppercaseCategory}_PERCENT`]:
+      Number(data[`usd_un${category}`]) * 100
   }))
 }
 
-export const newDataFormatExpensesAcrossElections = transactionsList =>
+// Income
```

Comment:
We need this comment

Reply:
Is for know by view although can add the name view and delete the comment